target/linux/ath79/image/common-mikrotik.mk

```diff
@@ -21,3 +21,8 @@ define Device/mikrotik_nand
   DEVICE_PACKAGES := nand-utils
   DEFAULT := n
 endef
+
+define Device/mikrotik_nand_yafut
```

Comment:
Shouldn't `Device/mikrotik_nand` profile be edited to get rid of the custom recipe that uses kernel2minor instead of adding new recipe that is not used?

Reply:
To me its not intrusive as currently no NAND images are generated, and since this tool finally allows them to be used again recipe should be updated.

@f00b4r0 Thoughts?

Reply:
> To me its not intrusive as **currently no NAND images are generated**, and since this tool finally allows them to be used again recipe should be updated.

That's a good point which I failed to consider: we no longer generate any NAND image, so there's no "switching over" per se. It would certainly speed up testing too; so I think we can do this. @kempniu sorry for the trouble :)

Reply:
@kempniu ath79 "supported" Mikrotik NAND devices are fairly consistent. I would think it's "okay" to switch them all over. Worst case scenario is a softbrick, these are "easy" to debrick (the only redeeming feature of RouterBoot I guess). So I'd suggest applying the change to all ath79 mikrotik NAND devices and see what happens. We're not touching release yet anyway, this can see some broader testing in master this way.

@robimarko makes a strong point that this would allow these devices to be properly supported again (if yafut does work reliably, which master testing will show).

Reply:
Fair enough.  All I'm trying to say is that (at this point) Yafut should be considered alpha-quality software that is capable of setting hardware on fire, causing PCB components to spontaneously emit toxic gases, tearing holes in spacetime and whatnot.  Your point about debricking being relatively easy makes sense to me, though, so does the part about master vs. release. I'll adjust the PR accordingly.

Reply:
> Yafut should be considered alpha-quality software that is capable of setting hardware on fire, causing PCB components to spontaneously emit toxic gases, tearing holes in spacetime and whatnot.

Thanks for the heads-up.
I'll let a junior engineer test this initially then over here.


Jokes aside,
I've got a ton of these boards (from very old to the latest revisions, different nand chips on some of them), so I'm happy to do extensive testing if it aids the cause.

Reply:
4a0bbce8471ebd0c15af43aa9f6ebd89c7f8d674 modifies the `mikrotik_nand` template instead of adding a separate one.  It also restores building MikroTik NAND images by defaults.  The commit message was updated accordingly.